prow/config/jobs/istio.io.yaml

```diff
@@ -0,0 +1,11 @@
+org: istio
+repo: istio.io
+image: gcr.io/istio-testing/build-tools:2019-08-31T04-48-30
+suppress_entrypoint: true
+branches:
+  - master
+  - release-1.2
```

Comment:
not 1.3 as well? because of the different branching approach I assume?

Reply:
Correct, there is no 1.3 branch at the moment.